test/form/samples/deprecated/interop-false-reexport/_expected/amd.js

```diff
@@ -2,7 +2,7 @@ define(['exports', 'external'], function (exports, external) { 'use strict';
 
 
 
-	exports.p = external.default;
+	exports.p = external;
```

Comment:
Is this a bug fix!?

Reply:
As a matter of fact yes, here, reexports behaved differently from regular imports where the `.default` was not added.